### ISSUE TITLE
Add Unibeautify to repository

### ISSUE THREAD

**Issue by micahlt** [ideoxan/ideoxan#5]
**Is your feature request related to a problem? Please describe.**
I noticed that the very detailed (it's awesome, by the way) `CONTRIBUTING.md` file contains an unfinished section on code formatting.  But it's hard to get contributors to all write code the same way.

**Describe the solution you'd like**
If we didn't want to deal with that, we could use Unibeautify for GitHub, a bot that automatically cleans up code.  That way people can write code how they want to and we'll still have a standard format.

**Describe alternatives you've considered**
Er, I suppose there are other code beautifiers for GitHub out there...

**Additional context**
👌

**Comment by Skyclo** [comment]
Yeah I do need to finish up the `CONTRIBUTING.md` file don't I... haha.

But anyways, I was just going to have some basic general guidelines for writing code (ie. for JavaScript stick to ES6 standards, keep code clear and concise, try to use comments, etc). However, I will look into Unibeautify!

**Comment by micahlt** [comment]
Ah, okay.  You've done a great job writing the file, though.  It's very thorough.

**Comment by Skyclo** [comment]
After toying around with Unibeautify, I have decided not to add it to the project. While it was easy for me to set up and everything worked, some of the code that the application modified couldn't be configured. I will keep this repository open in case you have any other suggestions for applications, bots, or anything else pertaining to the formatting of code.
